var/spack/repos/builtin/packages/racket/package.py

```diff
@@ -21,6 +20,10 @@ class Racket(Package):
 
     phases = ['configure', 'build', 'install']
 
+    def url_for_version(self, version):
+        url = "http://mirror.racket-lang.org/installers/{0}/racket-src.tgz"
+        return url.format(version.up_to(2))
```

Comment:
Does this mean that if a 8.3.1 gets releases the `url` is overwritten?

Reply:
The last patch release was in 2017, and they've been on a 3 month release cycle for minor versions since then (cf. https://mirror.racket-lang.org/installers/ )